index.html

```diff
@@ -25,24 +26,23 @@
 </head>
 
 <body class="main-block">
-  <header class="head-menu">
-    <nav class="nav-mobile">
-      <a class="logo" href="#">My Logo</a>
-      <a class="logo-ham" href="#"><i class="fa fa-bars"></i></a>
-    </nav>
-    <nav class="nav-desktop">
-      <a class="logo-desktop" href="#">My Logo</a>
-      <div class="nav-element-desktop">
-        <h3 class="elements">Portfolio</h3>
-        <h3 class="elements">About</h3>
-        <h3 class="elements">Contact</h3>
+  
+    <nav class="nav-bar">
+      <div class="nav-mobile">
+        <a class="logo logo-desktop" href="#">My Logo</a>
+        <a class="open-menu" href="#"><i class="fa fa-bars"></i></a>
+        <div class="close-menu hidden"><a href="#"><i class="fa fa-times"></i></a></div>
```

Comment:
[OPTIONAL] Kindly use the same event you use in the close button when pressing on any link as it now the scroll of the page disappear.